apps/cli/src/node/docker-compose-dev.yml

```diff
@@ -92,6 +92,7 @@ services:
             - 8080:8080
         environment:
             # s6-overlay
+            S6_STAGE2_HOOK: "/etc/s6-overlay/scripts/stage2-hook.sh"
```

Comment:
We need it into the docker-compose-dev because of the script content : 

```
else
    /command/s6-echo "host-runner: ENABLE_SUNODO_NODE is not set, skip the host mode..."
    sed -i '/host-runner/d' /usr/local/bin/is_ready
    rm /etc/traefik/conf.d/host-runner.yaml
fi
```